music21/musicxml/test_xmlToM21.py

```diff
@@ -1290,7 +1301,7 @@ def testHiddenRestImpliedVoice(self):
 
         self.assertEqual(len(MP.stream.voices), 2)
         self.assertEqual(len(MP.stream.voices[0].elements), 1)
-        self.assertEqual(len(MP.stream.voices[1].elements), 2)
+        self.assertEqual(len(MP.stream.voices[1].elements), 1)
```

Comment:
Would you mind updating the unit test name to avoid mentioning "hidden rests" now that we no longer create them? Maybe: gap?